src/components/dashboard/menu-table/popup-forms.js

```diff
@@ -433,11 +435,140 @@ const QRCodeForm = (props) => {
     );
 }
 
+const StyledUploadCSVModal = styled.div`
+    input[type="checkbox"] {
+        display: inline-block;
+    }
+
+    input[type="file"] {
+
+    }
+
+    .file-input {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 200px;
+        margin-bottom: 20px;
+        border: 2px dashed grey;
+        border-radius: 15px;
+
+        &:focus {
+            outline: none;
+        }
+    }
+
+    .error-msg {
+        color: red;
+    }
+    
+
+
+    p {
+        margin-left: 20px;
+        display: inline-block;
+    }
+`
+
+const UploadCSVModal = (props) => {
+    let { show, close, uploadCSV } = props
+    let [overwrite, setOverwrite] = useState(false)
+    let [content, setContent] = useState(null)
+    let [fileName, setFileName] = useState(null)
+    let [errorMessage, setErrorMessage] = useState(null)
+    
+    const onDrop = useCallback(acceptedFiles => {
+        acceptedFiles.forEach((file) => {
+            const reader = new FileReader()      
+            reader.onabort = () => console.error('file reading was aborted')
+            reader.onerror = () => {
+                console.error('file reading has failed')
+                setErrorMessage("Error uploading file")
+            }
+            reader.onload = () => {
+            // Do whatever you want with the file contents
+              const fileContent = reader.result
+              setContent(fileContent)
+              setFileName(file.path)
```

Comment:
I happened to drag in an empty file and the upload view did not respond to that. Nor did it prompt me about the empty file. Maybe we wanna take care of that? Like displaying file size maybe.